sdk/communication/Azure.Communication.Chat/src/Shared/CommunicationIdentifierSerializer.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json;
+
+namespace Azure.Communication
+{
+    internal class CommunicationIdentifierSerializer
```

Comment:
Can we move this file under `communication\Shared\src\` and statically link it?
As long as each individual library calls their identifier representation `CommunicationIdentifierModel` this will compile.

Alternatively we can make it public under Common with an interface `CommunicationIdentifierModelLike` and libraries add that interface to their partial `CommunicationIdentifierModel` class.

Reply:
I moved the serializer to Common.